pypesto/objective/priors.py

```diff
@@ -171,21 +171,25 @@ def get_parameter_prior_dict(index: int,
         index of the parameter in x_full
 
     prior_type: str
-        Prior is defined in LINEAR=untransformed parameter space! prior_type
-        can from {uniform, normal, laplace, logUniform, logNormal, logLaplace}
+        Prior is defined in LINEAR=untransformed parameter space,
+        unless it starts with "parameterScale"! prior_type
+        can be from {"uniform", "normal", "laplace", "logUniform", "logNormal",
+        "logLaplace", "parameterScaleUniform", "parameterScaleNormal",
```

Comment:
`"logUniform"` and `"logLaplace"` is described as not supported in the `_prior_densities` docstring